projects/laji/src/app/shared/pipe/includes-multi-state.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'includesMultiState'
+})
+export class IncludesMultiStatePipe implements PipeTransform {
+
+  transform(value: any, args: any, property: string, checkValue: string): boolean|undefined {
+    if (value.filter(v => v[property] === args[property]).length > 0) {
```

Comment:
if value can be anything calling filter on it without type checking is dangerous. Also the same `value.filter(v => v[property] === args[property]` is ran twice so using internal variable for that could be considered.

This methods signature could be shortened to `transform(value: unknown, option: SelectOptions): boolean|undefined` and target only the specific case of checkbox value.

Then the places where this is used could be changed from `includesMultiState:option:'id':'checkboxValue'` to `checkboxValue:option` (after the name change)